gdx/src/com/badlogic/gdx/maps/tiled/renderers/BatchTiledMapRenderer.java

```diff
@@ -186,7 +186,8 @@ public void renderObject (MapObject object) {
 	@Override
 	public void renderImageLayer (TiledMapImageLayer layer) {
 		final Color batchColor = batch.getColor();
-		final float color = Color.toFloatBits(batchColor.r, batchColor.g, batchColor.b, batchColor.a * layer.getOpacity());
+		final float color = Color.toFloatBits(batchColor.r * layer.getTintColor().r, batchColor.g * layer.getTintColor().g,
```

Comment:
If my suggested change for MapLayer.getTintColor() was applied, these repeated lines could multiply by layer.parent.getTintColor().r , .g , .b, etc.